.github/workflows/build_publish_reports.yml

```diff
@@ -0,0 +1,75 @@
+name: Build and Publish Reports
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - "bahmni-reports/**"
+      - ".github/workflows/build_publish_reports.yml"
+
+  workflow_dispatch:
+
+  repository_dispatch:
+    types: ["clinic-config-trigger"]
+
+env:
+  HELM_CHART_PATH: bahmni-reports/package/helm/
+
+jobs:
+  docker-build-publish:
+    name: Docker Build & Publish reports
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download clinic_config.zip
+        run: sh .github/download_artifact.sh clinic-config ClinicConfig ${{secrets.BAHMNI_PAT}} bahmni-reports
+      - name: Setting Artifact version
+        run: |
+          APP_VERSION=$(cat bahmni-reports/.appversion)
+          echo "ARTIFACT_VERSION=$(echo $APP_VERSION-${{github.run_number}})" >> $GITHUB_ENV
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Docker Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: bahmni-reports/package/docker/Dockerfile
+          push: true
+          tags: bahmniindiadistro/reports:${{env.ARTIFACT_VERSION}},bahmniindiadistro/reports:latest
```

Comment:
Should be bahmni-reports following the conventions

Reply:
Renaming of service names will be added as a backlog in ABDM board and picked up later